final_task/rss_parser/Classes/Novelty.py

```diff
@@ -0,0 +1,10 @@
+class Novelty:
```

Comment:
Take a look at `dataclasses.` This class fits to become `dataclass`

Reply:
Fixed, thanks.